shadowsocks/stream.go

```diff
@@ -262,7 +268,10 @@ type chunkReader struct {
 	aead cipher.AEAD
 	// Index of the next encrypted chunk to read.
 	counter []byte
-	buf     []byte
+	// Buffer for the uint16 size and its AEAD tag.  Made in init().
+	size []byte
```

Comment:
Rename to `payloadSizeBuf` to be clearer

Reply:
Done